src/helpers/update-pull-requests.ts

```diff
@@ -58,32 +54,23 @@ export async function updatePullRequests(context: Context) {
       if (isNaN(lastActivityDate.getTime()) || isPastOffset(lastActivityDate, requirements.mergeTimeout)) {
```

Comment:
It seems strange that we attempt to merge if the `lastActivityDate` is NaN, what's the reasoning behind this?